src/modules/picklist/__tests__/__snapshots__/pickListObjectComponent.js.snap

```diff
@@ -283,11 +276,8 @@ exports[`PickListComponent for objects Should mount 1`] = `
                     <td>
                       <span
                         className="pickListIcon">
-                        <FontAwesome
-                          name="folder">
-                          <span
-                            className="fa fa-folder" />
-                        </FontAwesome>
+                        <span
+                          className="icon icon-musitobject" />
```

Comment:
I didn't expect that the changes you have done to have impact on the snapshot files (?)